descriptors/ModuleDescriptor-template.json

```diff
@@ -8,7 +8,7 @@
     },
     {
       "id": "login",
-      "version": "6.0"
+      "version": "7.0"
```

Comment:
Can this be dual support for 6.0 and 7.0?

@craigmcnally what do you think? I thought the API that was removed was only used in one place in the UI which has since been removed as well?

Reply:
I was just looking at the code... I think actually that it might not even work with 7.0 as-is.  Confirming with @KaterynaSenchenko in slack

Reply:
fortunately I think we're all set.  It probably would have worked with "6.0 7.0"